config/routes.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
   devise_for :users, controllers: { omniauth_callbacks: "users/omniauth_callbacks" }
   root "quotes#index"
+  get "quotes/best", to: "quotes#best"
```

Comment:
This is better done as a "collection" route.

```ruby
 resources :quotes do
   collection do
    get :best
   end
   resources :votes, only: [:create, :update, :destroy]
 end
```

Probably time to read: http://guides.rubyonrails.org/routing.html